### ISSUE TITLE
With setting the device to cpu in the hyperparameters used for training in the sst2 prompt tuning example

### ISSUE THREAD

**Issue by jeromemassot** [bigscience-workshop/petals#158]
Hi Big-Science team,

quick question: Why is the device set to 'CPU' in these training hyperparameters?

Especially when the notebook recommends using a GPU.

MODEL_NAME = "bigscience/bloom-petals" # select model you like
NUM_PREFIX_TOKENS = 16
DEVICE = 'cpu'
BATCH_SIZE = 16
LR = 1e-2
WEIGHT_DECAY = 0.0
NUM_EPOCHS = 3
SEED = 42
MODEL_MAX_LENGTH = 64
TUNING_MODE = 'ptune' # choose between ['ptune', 'deep_ptune'] 

Thanks

Best regards

Jerome

**Comment by artek0chumak** [comment]
Hi!
Thank you for the issue. It's miss typo that was not caught by our team. It will be changed to "cuda", as the notes suggest.
